### ISSUE TITLE
VoiceOver users can't read conversations

### ISSUE THREAD

**Issue by fcnjd** [element-hq/element-x-ios#4696]
### Steps to reproduce

1. Turn on VoiceOver and open a private chat or room.
2. Only the header buttons (back, call) and the bottom once (attach media, keyboard) are visible.
3. The whole conversation body is not reachable, neither via swipe nor tap.

#### What did you expect?
I expect VoiceOver to read out the chat messages, both being able to use the swipe gestures to focus previous and next message, as well as directly selecting a message.

#### What happened instead?
Because of this bug, it is impossible for any blind iOS user to read a chat with Element X, which is breaking the whole app.

### Your phone model

iPhone 15 Pro

### Operating system version

iOS 26.1

### Application version

25.10.2

### Homeserver

private and not relevant here

### Will you send logs?

No

**Comment by pixlwave** [comment]
Thanks for reporting this @fcnjd, it looks we have a major regression. Video of what happens (turn on sound) 👇

https://github.com/user-attachments/assets/f9d14dc6-82f0-4060-b4cd-c5cd4106f84d

**Comment by pixlwave** [comment]
This will be fixed in 25.11.0, which should be available on TestFlight shortly and all things being well, on the App Store on Monday. Thanks again for reporting and sorry that the core functionality of the app was broken for you.

**Comment by fcnjd** [comment]
Thank you very much for the quick update, it's nice to see that you're taking such issues serious. I'm looking forward to the new version.
